### PR TITLE
Release v1.1

### PR DESCRIPTION
**Pull request by mts299**
# Release v1.1
## scope 
- [x] updates to RAWACF bug 
- [x] updates to setup.py install to grab head of the git repos 
- [x] ACF plotting 
- [x] Borelais v 5.0 reading 
- [x] Deprecation warnings 

## Testing 

1. can you install it with no errors (please use `python3 setup.py install` to avoid #59 )
2. Test on reading in RAWACF files and plot some ACFs #43 (testing code here)
3. Test if you get the `PendingDeprecationWarning` with any reading abilities  
4. Test reading in new Borealis files contact @mardet987 for files
5. Check zenodo `.zenodo.json` file that your contact info is correct

## Done
Once you are done please approve the PR 

### When to merge
Once there are 3 approvals or 2 weeks of testing.

### CODE REVIEW

File: pydarn/io/borealis/borealis_convert.py [hunk 1]
```diff
@@ -202,9 +218,22 @@ def __init__(self, borealis_filename: str, borealis_filetype: str,
...
+                self._borealis_slice_id = int(borealis_slice_id)
+            else:
+                raise borealis_exceptions.BorealisStructureError(
+                    'The slice_id could not be found in the file: Borealis '
+                    'files produced before Borealis v0.5 must provide the '
+                    'slice_id value to the BorealisConvert class.') from e
+
         self._sdarn_dmap_records = {}
         self._sdarn_dict = {}
+        self._scaling_factor = scaling_factor
```
Comment (mardet987) [inline pydarn/io/borealis/borealis_convert.py:236]: Was doing some testing and came across an error that was not very clear for the user and would be solved by this suggestion!

```suggestion
        self._scaling_factor = int(scaling_factor)
```

**Comment by mts299** [comment]
Hey, everyone please check out this release before it goes out.

**Comment by mardet987** [comment]
I'm getting an error:

**Comment by mts299** [comment]
@mardet987 Fixed the bug, please try your test again :)

**Comment by mts299** [comment]
Updated documentation to include deprecation warnings: https://pydarn.readthedocs.io/en/release-v1.1/

**Comment by carleyjmartin** [comment]
For anyone testing the ACF plotting: it's not ...ACF.plot() anymore it's ..ACF.plot_acfs() in the test code.

**Comment by mts299** [comment]
Thanks, @carleyjmartin trying to write up some documentation on that :p

**Comment by mts299** [comment]
Changed the plotting of ACF's blanked lags back to True upon request of @asreimer as it is a diagnostic tool for engineers.

**Comment by mts299** [comment]
Its been a couple of weeks and 2 approvals with approval via private message, I will be merging this code

**Pull request merged**